Source/source/GPS.cpp

```diff
@@ -29,9 +27,10 @@ bool poll(string *message) {
 }
 
 //send in a NMEA string, get organized gps_data
-gps_data decode(string raw) { 
-	gps_data data = {};
+void decode(string raw) { 
+	data = {};
 	minmea_sentence_gga ggaFrame;
+	minmea_sentence_zda zdaFrame;
```

Comment:
Is this used anywhere? should there be a minmea_parse_zda call below?

Reply:
Ah this code was removed by accident. I'll re-implement that now